.kokoro/linkage-monitor.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Looks like this file needs to be made executable

Reply:
Fixed!
https://github.com/googleapis/google-http-java-client/pull/719/commits/ba63ad29b4efdfa7762b1986d312929633b990f0